geps/gep-713/index.md

```diff
@@ -3,1155 +3,1192 @@
+
+Structured data cannot be used because there is no way of knowing what the structure will be beforehand. This suggests a need to use unstructured data for representing the main body of the arbitrary policy objects. Practically, this will have to be a string representation of the YAML form (or JSON, equivalently) of the body of the policy object (absent the metadata part of every Kubernetes object).
+
+Metaresources and Policy Attachment does not mandate anything about the design of the object's top level except that it must be a Kubernetes object, so the only possible thing to rely upon here is the presence of the Kubernetes metadata elements: `apiVersion`, `kind`, and `metadata`.
+
+Therefore, a string representation of the rest of the file is likely the best that can be done here.
+
+#### Fanout status update problems
+
+The fanout problem is that, when an update takes place in a single object (a policy, or an object with a policy attached), an implementation may need to update _many_ objects if it needs to place details of what policy applies, or what the resultant set of policies is on _every_ object.
+
+Historically, this is a risky strategy and needs to be carefully applied, as it's an excellent way to create apiserver load problems, which can produce a large range of bad effects for cluster stability.
+
+This does not mean that nothing at all that affects multiple objects can be done, but that careful consideration of what information is stored in status, so that _every_ policy update does not require a corresponding status update, is advised.
+
+## Current use of policies
+
+### Implementations
+
+These are a few known implementations of policies in compliance with this GEP.
+
+Users should refer to the official documentation from each implementation for for more up to date information.
+
+#### Gateway API (core)
+
+Gateway API defines two kinds of Direct policies, both for augmenting the behavior of Kubernetes `Service` resources:
+
+| Policy kind               | Description                                                                                                     | Target kinds    | Merge strategies | Policy class |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------- |---------------- | ---------------- | ------------ |
+| **BackendTLSPolicy**      | TLS configuration of the connection from the Gateway to a backend pod (set of pods) via the Service API object. | Service, _Port_ | None             | Direct       |
+| **XBackendTrafficPolicy** | Configuration for how traffic to a target backend should be handled (retries and session persistence)           | _Port_          | None             | Direct       |
+
+#### Envoy Gateway
+
+<small>https://gateway.envoyproxy.io/docs/api/extension_types/</small>
```

Comment:
This should probably be a link. Same comment applies to all links below. See https://deploy-preview-3609--kubernetes-sigs-gateway-api.netlify.app/geps/gep-713/#envoy-gateway